src/Mod/TechDraw/Gui/ViewProviderPage.cpp

```diff
@@ -209,8 +205,9 @@ bool ViewProviderPage::onDelete(const std::vector<std::string>&)
             "Std_Delete",
             "The page is not empty, therefore the\nfollowing referencing objects might be lost:");
         bodyMessageStream << '\n';
-        for (auto ObjIterator : objs)
+        for (auto ObjIterator : objs) {
             bodyMessageStream << '\n' << QString::fromUtf8(ObjIterator->Label.getValue());
+        }

```

Comment:
```suggestion
        for (const auto& obj : objs) {
             bodyMessageStream << '\n' << QString::fromUtf8(ObjIterator->Label.getValue());
        }
```
Important part: `const auto&`

Less important part: note that iterator is a class type(?) in C++. Naming might be confusing.